scripts/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "internal-scripts",
+  "version": "0.0.0-development",
```

Comment:
This gets rid of that annoying warning saying that scripts doesn't have a version